docs/content/Dialog2.mdx

```diff
@@ -89,8 +89,8 @@ By default, the Dialog component implements the design and interactions defined
 | footerButtons | `DialogButtonProps[]`                          |            | Optional. Specify buttons that will be rendered in the footer of the dialog.                                                                                                                                                                                            |
 | onClose       | `(gesture: 'close-button' │ 'escape') => void` |            | Required. This method is invoked when a gesture to close the dialog is used (either an Escape key press or clicking the "X" in the top-right corner). The gesture argument indicates the gesture that was used to close the dialog (either 'close-button' or 'escape'). |
 | role          | `"dialog" │ "alertdialog"`                     | `"dialog"` | The ARIA role given to the dialog element. More info: [dialog](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal), [alertdialog](https://www.w3.org/TR/wai-aria-practices-1.1/#alertdialog)                                                                    |
-| width         | `"sm" │ "md" │ "lg" │ "xl"`                    | `"xl"`     | The fixed width of the dialog.                                                                                                                                                                                                                                          |
-| height        | `"sm" │ "lg" │ "auto"`                         | `"auto"`   | The fixed height of the dialog, or, auto to adjust the height based on its contents.                                                                                                                                                                                    |
+| width         | `"small" │ "medium" │ "large" │ "xLarge"`                    | `"xLarge"`     | The fixed width of the dialog.                                                                                                                                                                                                                                          |
```

Comment:
Chatted with @yaili about this and we prefer `xlarge` (with a lowercase `l`). What do you think?

```suggestion
| width         | `"small" │ "medium" │ "large" │ "xlarge"`                    | `"xlarge"`     | The fixed width of the dialog.                                                                                                                                                                                                                                          |
```